yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/resources/ResourceCreationContainer.java

```diff
@@ -0,0 +1,57 @@
+package com.yoti.api.client.docs.session.create.resources;
+
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Container for creating resources at session creation.
+ */
+public class ResourceCreationContainer {
+
+    @JsonProperty(Property.APPLICANT_PROFILE)
+    private final Map<String, Object> applicantProfile;
+
+    private ResourceCreationContainer(Map<String, Object> applicantProfile) {
+        this.applicantProfile = applicantProfile;
+    }
+
+    public static ResourceCreationContainer.Builder builder() {
+        return new ResourceCreationContainer.Builder();
+    }
+
+    /**
+     * The applicant profile resource to be created.
+     *
+     * @return the applicant profile fields
+     */
+    public Map<String, Object> getApplicantProfile() {
+        return applicantProfile;
+    }
+
+    public static final class Builder {
+
+        private Map<String, Object> applicantProfile;
+
+        private Builder() {}
+
+        public Builder withApplicantProfile(Map<String, Object> applicantProfile) {
+            this.applicantProfile = applicantProfile;
+            return this;
+        }
+
+        public ResourceCreationContainer build() {
+            return new ResourceCreationContainer(applicantProfile);
```

Comment:
it's good practice to have the class contructor with the builder as arg when using Builders

```
private ResourceCreationContainer(Builder builder) {
        applicantProfile = builder.applicantProfile;
}

public ResourceCreationContainer build() {
            return new ResourceCreationContainer(this);
}
```

Reply:
This isn't how it's done in the rest of the SDK, so I'd like to keep it similar as possible.  If this is something you think would be good to do, probably something worth doing in the next major and updating everything.

Reply:
I'm afraid the next major release requires a nuke :)